front-end/tailwind.config.js

```diff
@@ -9,8 +9,12 @@ module.exports = {
     colors: {
       transparent: "transparent",
       current: "currentColor",
+      "black": "#000000",
```

Comment:
```suggestion
      "black": "#1B1B1B",
```
I'm a bit wary of pure black, this one is similar enough to a dark black, but not completely 000000. Too contrast-y colours like pure white and pure black are straining on the eye, so ppl with not so great vision or just ppl who look to long at the colours start getting hurt-ish from it

Reply:
Discussed this on discord, but I'm keeping this here as the black is only used to reset some of the text in the default menu to black, which should be #000000. Black is not used to actually fill in any part of the component. But I agree with what u said in that it would've been too harsh if I had used this black in more than that.